operator/src/main/java/oracle/kubernetes/operator/rest/RestConfigImpl.java

```diff
@@ -4,26 +4,21 @@
 
 package oracle.kubernetes.operator.rest;
 
-import java.io.File;
-import java.nio.file.Files;
-import java.nio.file.Paths;
 import java.util.Collection;
+import oracle.kubernetes.operator.OperatorConstants;
+import oracle.kubernetes.operator.TuningParameters;
 import oracle.kubernetes.operator.logging.LoggingFacade;
 import oracle.kubernetes.operator.logging.LoggingFactory;
 import oracle.kubernetes.operator.rest.backend.RestBackend;
 
 /** RestConfigImpl provides the WebLogic Operator REST api configuration. */
-public class RestConfigImpl implements RestConfig {
+public class RestConfigImpl implements RestConfig, OperatorConstants {
```

Comment:
Are you implementing an interface in order to import constants? That's a very old practice, and should be avoided. Instead, you can statically import them into a class if you want to access them without specifying the interface on every reference.

Reply:
I'm working on a different change to reuse the RestConfig.  Might want to wait on the review.  Thank you!